csrc/ir/internal_nodes.h

```diff
@@ -2255,4 +2255,37 @@ class NVF_API CatOp : public Expr {
   Val* getPred(int input_idx) const;
 };
 
+//! Matmul Operator to be expression evaluated without decomposition.
+class NVF_API MatmulOp : public Expr {
```

Comment:
https://github.com/NVIDIA/Fuser/blob/main/doc/dev/visibility.md#should-i-mark-my-new-method-or-class-as-nvf_api

```suggestion
class MatmulOp : public Expr {
```